lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -287,6 +290,17 @@ public Map<String, ContraIndicatorScore> getContraIndicatorScoresMap() {
         }
     }
 
+    public Map<String, ContraIndicatorMitigation> getCiMitConfig() throws ConfigException {
+        final TypeReference<HashMap<String, ContraIndicatorMitigation>> typeRef =
+                new TypeReference<>() {};
+        final String ciMitConfig = getSsmParameter(ConfigurationVariable.CIMIT_CONFIG);
+        try {
+            return objectMapper.readValue(ciMitConfig, typeRef);
```

Comment:
I think, can be write directly new TypeReference<>() {} and getSsmParameter(ConfigurationVariable.CIMIT_CONFIG) in objectMapper.readValue().

Reply:
Amended